src/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -54,10 +60,14 @@
   private JButton m_ruleBook;
   private JButton m_donate;
   private JButton m_about;
+
   private final SetupPanelModel m_model;
+  private final MapDownloadController mapDownloadController;
 
   public MetaSetupPanel(final SetupPanelModel model) {
-    m_model = model;
+    this.m_model = model;
+    this.mapDownloadController = new MapDownloadController();
```

Comment:
A new download controller added here so we can remove " DownloadMapDialog.downloadGames(this);", and replace it with "mapDownloadController.openDownloadMapScreen(parentWindow);"